app/models/setting/content.rb

```diff
@@ -9,6 +9,7 @@ def self.load_defaults
 
     download_policies = proc { Hash[::Runcible::Models::YumImporter::DOWNLOAD_POLICIES.map { |p| [p, p] }] }
     proxy_download_policies = proc { Hash[::SmartProxy::DOWNLOAD_POLICIES.map { |p| [p, p] }] }
+    dependency_solving_options = proc { Hash[['conservative', 'greedy'].map { |p| [p, p] }] }
```

Comment:
Wonder if this is clearer
```
dependency_solving_options = proc { {'conservative': 'conservative', 'greedy': 'greedy']} }
```

Reply:
though still need to try this one

Reply:
@parthaa I added a method to handle converting an array to hash with a key/value of each element in the array